wikidatarefisland/pipes/pipe_one.py

```diff
@@ -0,0 +1,44 @@
+import json
+import jsonlines
+from functools import reduce
+from itertools import chain
+from wikidatarefisland.pipes import statement_filters
+
+
+class PipeOne():
```

Comment:
Gotcha, I was trying to capture what it does in the sense of this project and was thinking that the pipe/side service names might be our "common language" given that's what I noticed us using during meetings and in tickets.

Would you see any merit in keeping the Pipe names as the entrypoint and trying to extract the component parts to names that more closely mirror what they do?

For me conceptually the thing we're building has been talked about with pipe names for so long that they "mean more" to me than ItemAnalyzor or ItemDataExtractor or something.

Reply:
One thing we can do is to have interface of Pipe and make this class implement it. I like that a lot. Also having all of those in the namespace of "pipes" would help plus writing in the docstring that this is pipe one (and in the README saying that the class ItemDataExtractor would be great).

I agree that pipe1 is more intuitive to us (count me in too) but I don't think it would be easily maintainable for others who want to develop this codebase,